package.json

```diff
@@ -56,6 +56,7 @@
     "lint": "eslint . --cache",
     "docs": "node scripts/generate-readme.js",
     "integration-test-cleanup": "node scripts/integration-test-cleanup.js",
+    "local-integration-test": "jest --maxWorkers=5 local-suite",
```

Comment:
A suggeston: I would probably name it `package-integration-test` and those that deploy projects to AWS `deploy-integration-test`